### PR TITLE
Add scikit optimize wrapper

### PR DESCRIPTION
**Pull request by yoshihikoueno**
This change adds a wrapper to expose this cma-es optimizer in skopt-compatible API, which enables us to utilize many convenient tools and functions built for scikit optimize.

### CODE REVIEW

**Comment by nikohansen** [comment]
Thanks, that's a very nice enhancement! I'll have a look at it ASAP!

File: cma/wrapper.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        model.logger.add()
+        if verbose: model.disp()
+
+        xi += new_xi_denorm
+        yi += new_yi
+        results = create_result(xi, yi)
+        for f in callback: f(results)
+
+    results = create_result(xi, yi, space)
+    results.cma_logger = model.logger
```
Comment (nikohansen) [inline cma/wrapper.py:90]: I didn't test it yet, but I think that we need here `model.logger.load()` to get the data read in before the folder is deleted.

File: setup.py [hunk 2]
```diff
@@ -51,6 +51,6 @@
       ],
       keywords=["optimization", "CMA-ES", "cmaes"],
       packages=["cma", "cma.utilities"],
-      requires=["numpy"],
+      install_requires=["numpy", "scikit-optimize"],
```
Comment (nikohansen) [inline setup.py:54]: I found that `setuptools` has an [`extras_require`](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) keyword, which seem we should move the scikit requirement in.

**Comment by yoshihikoueno** [comment]
@nikohansen Thank you for the very quick review! I've fixed the issues.

**Pull request merged**